src/frontend/index.tsx

```diff
@@ -1,25 +1,38 @@
 import 'iframe-resizer/js/iframeResizer.contentWindow';
 
 import { Grommet } from 'grommet';
+import jwt_decode from 'jwt-decode';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { IntlProvider } from 'react-intl';
+import { addLocaleData, IntlProvider } from 'react-intl';
+import en from 'react-intl/locale-data/en';
+import es from 'react-intl/locale-data/es';
+import fr from 'react-intl/locale-data/fr';
 import { Provider } from 'react-redux';
 
 import { AppRoutesConnected } from './components/AppRoutesConnected/AppRoutesConnected';
 import { appData } from './data/appData';
 import { bootstrapStore } from './data/bootstrapStore';
+import translatedMessages from './translations/translation.json';
```

Comment:
This (along with the `react-intl/locale-data` is fine for now but if we support more languages or want to optimize payloads we'll need to load only the locale we need.